libnmstate/__init__.py

```diff
@@ -36,3 +41,9 @@
     "schema",
     "PrettyState",
 ]
+
+
+def __version__():
+    with open(os.path.join(ROOT_DIR, "VERSION")) as f:
+        version = f.read().strip()
```

Comment:
change this to `__version__ = ...` and move out of the function. Also there should be a test for this code that at least accesses the property without verifying its value. Verifying the value would, be good, too.